config.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
perhaps ignorantly i am confused by this file.  is this a sample config file?  it seems to be config info for testing.  i don't know if that is proper. @Foxcapades

Reply:
@steve-fischer-200 I think these are the sequences that are loaded into memory an indexed by the service: https://github.com/VEuPathDB/service-sequence-retrieval/pull/1/files#diff-4e416d8b17c2c7e05070d9d4cf7870cbbb182b63f1f71a805091874c38fd8de6R44. Looks like for the POC it's just a couple of test sequences

Reply:
I don't know what's a good jenkins for this but in deployment, overwrite this config.json with a proper config.json.

The .jar will expect a config.json in the current working directory.